### DISCUSSION TITLE
Ngrx/store with Angular 16 Hydration

### DISCUSSION THREAD

**Discussion by AbrahamGreenman** [ngrx/platform#3954] [General]
I was wondering if any extra steps are needed for hydration, or if can be kept as is. It would seem nothing needs to be changed as hydration preserves DOM but just wanted to open a conversation. Thank you

**Comment by brandonroberts** [comment]
No extra steps are needed for hydration as far as NgRx Store is concerned. If you are using SSR to persist/hydrate the store and you have some sample code to share, that would be useful. 

Maybe we can look at providing some integration there.

**Reply by AbrahamGreenman** [reply]
ok, i will think on it. Using standard:

```
providers: [
    { provide: APP_ID, useValue: 'server-app' },
    provideClientHydration()
  ],
```

It would seem this can be closed then as all systems go.
